README.md

```diff
@@ -107,6 +107,14 @@ Bluepill only works with **Xcode 9.1**. If you're looking for old Xcode support,
 
 Bluepill was inspired by [parallel iOS test](https://github.com/plu/parallel_ios_tests) and Facebook’s [xctool](https://github.com/facebook/xctool) and [FBSimulatorControl](https://github.com/facebook/FBSimulatorControl). The Bluepill icon was created by [Maria Iu](https://www.linkedin.com/in/mariaiu/).
 
+## Fastlane Support
+
+If you're interested in running bluepill using [Fastlane](https://github.com/fastlane/fastlane), you're in luck! There are a few open-sourced plugins available for use with Fastlane. Here's the current list:
+
+- [Redpill](https://github.com/omniprojects/fastlane-plugin-redpill) - The most up-to-date plugin; it's is maintained by [Omni](https://github.com/omniprojects) and offers a full set of options to hook into bluepill. As of this writing, Redpill is compiled against XCode 9.1, and intended for use with [CircleCI](https://circleci.com). It will be updated with newer versions of xCode when CircleCI supports them.
```

Comment:
minor replace all XCode & xCode with Xcode, also bluepill -> Bluepill :)

Reply:
Got it! Doing that now...